tests/examples/opinions/united_states/nh_p_example.compare.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "case_dates": "2023-06-29",
+    "case_names": "In re E.R. In re H.R.",
```

Comment:
this shouldn't be one sentence, it needs to at least have a semicolon to separate the opinions.

Reply:
This is done automatically by `AbstractSite._clean_attributes`. It uses `harmonize` on case names. The source data is indeed `"In re E.R.; In re H.R."`

```python
from juriscraper.lib.string_utils import harmonize
harmonize("In re E.R.; In re H.R.")

# Out[2]: 'In re E.R. In re H.R.'
```

Reply:
Troubling.

Reply:
Looks like we just strip out semicolons in 
`juriscraper.lib.string_utils.clean_string`

I do not think we should just remove semicolon's from text - 

    # Get rid of weird punctuation
    s = s.replace("*", "").replace("#", "").replace(";", "")

I think we should drop `).replace(";", "")`

@mlissner do you have any feelings about this.

Reply:
@flooie  Should we move this change to another issue / PR? Changing `clean_string` will break other test cases, and seems awkward to do it in this small PR for `nh`. This is just a sample of the breaking changes, it exceeded the line limit of my terminal
 
```
======================================================================
FAIL: test_scrape_opinion_fed_app_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ca2_u_example.compare.json', module_string='juriscraper.opinions.united_states.federal_appellate.ca2_u')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[294 chars]s': '10-4934-cv (L) 10-4965-cv (Con)', 'case_name_shorts': ''} != {'cas[294 chars]s': '10-4934-cv (L); 10-4965-cv (Con)', 'case_name_shorts': ''}
  {'blocked_statuses': False,
   'case_dates': '2011-10-20',
   'case_name_shorts': '',
   'case_names': 'Chem One, Ltd. v. M/V Rickmers Genoa',
   'date_filed_is_approximate': False,
-  'docket_numbers': '10-4934-cv (L) 10-4965-cv (Con)',
+  'docket_numbers': '10-4934-cv (L); 10-4965-cv (Con)',
?                                   +

   'download_urls': '/decisions/isysquery/f3512627-eddb-42db-af2f-b881a9f90d1f/90/doc/10-4934_opn.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_fed_bankr_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/bap9_example.compare.json', module_string='juriscraper.opinions.united_states.federal_bankruptcy.bap9')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[275 chars]s': 'CC-19-1039-STaF CC-19-1040-STaF', 'case_name_shorts': ''} != {'cas[275 chars]s': 'CC-19-1039-STaF; CC-19-1040-STaF', 'case_name_shorts': ''}
  {'blocked_statuses': False,
   'case_dates': '2020-02-11',
   'case_name_shorts': '',
   'case_names': 'In re: Maggie Liu',
   'date_filed_is_approximate': False,
-  'docket_numbers': 'CC-19-1039-STaF CC-19-1040-STaF',
+  'docket_numbers': 'CC-19-1039-STaF; CC-19-1040-STaF',
?                                    +

   'download_urls': 'http://cdn.ca9.uscourts.gov/datastore/bap/2020/02/11/Liu '
                    '19-1039 and 19-1040.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_fed_special_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/uscfc_example.compare.json', module_string='juriscraper.opinions.united_states.federal_special.uscfc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[436 chars]gment granting, in part,[129]Cross-Motion for [82 chars]: ''} != {'cas[436 chars]gment; granting, in part,[129]Cross-Motion for[83 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2014-05-09',
   'case_name_shorts': '',
   'case_names': 'Principal Life Insurance Company and Subsidiaries v. United '
                 'States',
   'date_filed_is_approximate': False,
   'docket_numbers': '1:07-cv-00006',
   'download_urls': 'https://ecf.cofc.uscourts.gov/cgi-bin/show_public_doc?2007cv0006-180-0',
   'judges': 'Francis M. Allegra',
   'precedential_statuses': 'Published',
-  'summaries': 'REPORTED OPINION denying Motion for Partial Summary Judgment '
+  'summaries': 'REPORTED OPINION denying Motion for Partial Summary Judgment; '
?                                                                            +

                'granting, in part,[129]Cross-Motion for Partial Summary '
                'Judgment. Signed by Judge Francis M. Allegra.'}

======================================================================
FAIL: test_scrape_opinion_fed_special_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/uscfc_example_2.compare.json', module_string='juriscraper.opinions.united_states.federal_special.uscfc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[372 chars]gment finding as moot [29] Motion to Dismiss -[297 chars]: ''} != {'cas[372 chars]gment; finding as moot [29] Motion to Dismiss [299 chars]: ''}
Diff is 2073 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_fed_special_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/uscfc_u_example.compare.json', module_string='juriscraper.opinions.united_states.federal_special.uscfc_u')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[413 chars]b)(6) dismissing as moot [15] Motion for Summa[194 chars]: ''} != {'cas[413 chars]b)(6); dismissing as moot [15] Motion for Summ[196 chars]: ''}
Diff is 1024 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_fed_special_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/fisc_example.compare.json', module_string='juriscraper.opinions.united_states.federal_special.fisc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[152 chars] 2020 and Partial Response to the Court's Orde[318 chars]: ''} != {'cas[152 chars] 2020; and Partial Response to the Court's Ord[319 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2021-07-16',
   'case_name_shorts': '',
   'case_names': "Supplemental Response to the Court's Order Dated December 17, "
-                '2019 and Corrected Opinion and Order Dated March 5, 2020 and '
+                '2019 and Corrected Opinion and Order Dated March 5, 2020; and '
?                                                                         +

                 "Partial Response to the Court's Order Dated April 3, 2020",
   'date_filed_is_approximate': False,
   'docket_numbers': 'Misc. 19-02',
   'download_urls': 'https://www.fisc.uscourts.gov/sites/default/files/FISC%20Misc%2019%2002%20Supplemental%20Response%20Jluly%2016%202021.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/arizctapp_div_2_example.compare.json', module_string='juriscraper.opinions.united_states.state.arizctapp_div_2')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[512 chars]m. P. A.R.S. § 13-3967 or the constitutional r[90 chars]: ''} != {'cas[512 chars]m. P.; A.R.S. § 13-3967; or the constitutional[92 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2005-05-10',
   'case_name_shorts': '',
   'case_names': 'Renee Antonio Fragoso v. the State of Arizona',
   'date_filed_is_approximate': False,
   'docket_numbers': '2 CA-SA 2005-0001',
   'download_urls': 'https://www.appeals2.az.gov/Decisions/SA20050001_Opinion.pdf',
   'precedential_statuses': 'Published',
   'summaries': 'Does a trial court's imposition of "cash-only" bail as a '
                "condition of a defendant's pretrial release violate Arizona's "
                'pertinent criminal rules or statute, Rules 7.1 and 7.3, Ariz. '
-               'R. Crim. P. A.R.S. § 13-3967 or the constitutional right to '
+               'R. Crim. P.; A.R.S. § 13-3967; or the constitutional right to '
?                           +                 +

                'bail "by sufficient sureties," Ariz. Const. art. II, § 22(A)?'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ark_example.compare.json', module_string='juriscraper.opinions.united_states.state.ark')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[51 chars]tuart James Greenhaw Elizabeth Jessup Amber Wo[511 chars]alse} != {'cas[51 chars]tuart; James Greenhaw; Elizabeth Jessup; Amber[520 chars]alse}
Diff is 1619 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/arkctapp_example.compare.json', module_string='juriscraper.opinions.united_states.state.arkctapp')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[73 chars]r, Md And Freeway Surgery Center, LLC', 'downl[284 chars]alse} != {'cas[73 chars]r, Md; And Freeway Surgery Center, LLC', 'down[285 chars]alse}
  {'blocked_statuses': False,
   'case_dates': '2024-04-03',
   'case_name_shorts': '',
-  'case_names': 'Sherri Reggans v. Scott Schlesinger, Md And Freeway Surgery '
+  'case_names': 'Sherri Reggans v. Scott Schlesinger, Md; And Freeway Surgery '
?                                                        +

                 'Center, LLC',
   'citations': '2024 Ark. App. 227',
   'date_filed_is_approximate': False,
   'docket_numbers': '',
   'download_urls': 'https://opinions.arcourts.gov/ark/courtofappeals/en/522663/1/document.do',
   'per_curiam': False,
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/calag_example_2.compare.json', module_string='juriscraper.opinions.united_states.state.calag')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[599 chars]ittee (b) an appointed public member of the SA[2312 chars]: ''} != {'cas[599 chars]ittee; (b) an appointed public member of the S[2314 chars]: ''}
Diff is 4031 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/dc_example_2.compare.json', module_string='juriscraper.opinions.united_states.state.dc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[77 chars]pital Sedgwick Claims Management Services, Inc[208 chars]: ''} != {'cas[77 chars]pital; Sedgwick Claims Management Services, In[209 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2017-05-04',
   'case_name_shorts': '',
-  'case_names': 'Jacqueline Dent v. DOES/Providence Hospital Sedgwick Claims '
+  'case_names': 'Jacqueline Dent v. DOES/Providence Hospital; Sedgwick Claims '
?                                                            +

                 'Management Services, Inc.',
   'date_filed_is_approximate': False,
   'docket_numbers': '14-AA-527',
   'download_urls': '/internet/documents/14-AA-527.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/dc_example_3.compare.json', module_string='juriscraper.opinions.united_states.state.dc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[44 chars] K.M. M.M.', 'download_urls': '/sites/default/[182 chars]: ''} != {'cas[44 chars] K.M.; M.M.', 'download_urls': '/sites/default[183 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2017-08-10',
   'case_name_shorts': '',
-  'case_names': 'In re K.M. M.M.',
+  'case_names': 'In re K.M.; M.M.',
?                           +

   'date_filed_is_approximate': False,
   'docket_numbers': '15-FS-1417',
   'download_urls': '/sites/default/files/2017-08/15-FS-1417.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/fla_example.compare.json', module_string='juriscraper.opinions.united_states.state.fla')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[74 chars]urray Gerald Delane Murray v. Julie L. Jones, [293 chars]: ''} != {'cas[74 chars]urray; Gerald Delane Murray v. Julie L. Jones,[295 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2018-12-20',
   'case_name_shorts': '',
-  'case_names': 'State of Florida v. Gerald Delane Murray Gerald Delane Murray '
+  'case_names': 'State of Florida v. Gerald Delane Murray; Gerald Delane Murray '
?                                                         +

-                'v. Julie L. Jones, etc. and Gerald Delane Murray v. State of '
+                'v. Julie L. Jones, etc.; and Gerald Delane Murray v. State of '
?                                        +

                 'Florida',
   'date_filed_is_approximate': False,
   'docket_numbers': 'SC17-707, SC18-560 & SC18-334',
   'download_urls': '/content/download/425242/4583114/file/sc17-707.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/fladistctapp_3_example.compare.json', module_string='juriscraper.opinions.united_states.state.fladistctapp_3')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[291 chars]irmed remanded with instructions.', 'docket_nu[62 chars]alse} != {'cas[291 chars]irmed; remanded with instructions.', 'docket_n[63 chars]alse}
  {'blocked_statuses': False,
   'case_dates': '2024-08-28',
   'case_name_shorts': '',
   'case_names': 'Deymus Ramos Gonzalez v. the State of Florida',
   'date_filed_is_approximate': False,
-  'dispositions': 'Affirmed remanded with instructions.',
+  'dispositions': 'Affirmed; remanded with instructions.',
?                           +

   'docket_numbers': '3D2021-1445',
   'download_urls': 'https://3dca.flcourts.gov/pre_opinion_content_download/2439726',
   'per_curiam': False,
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/fladistctapp_6_example.compare.json', module_string='juriscraper.opinions.united_states.state.fladistctapp_6')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[284 chars]irmed Conflict Certified.', 'docket_numbers': [54 chars]alse} != {'cas[284 chars]irmed; Conflict Certified.', 'docket_numbers':[55 chars]alse}
  {'blocked_statuses': False,
   'case_dates': '2024-08-23',
   'case_name_shorts': '',
   'case_names': 'William A. Julia v. Melissa Ramos-Baez',
   'date_filed_is_approximate': False,
-  'dispositions': 'Affirmed Conflict Certified.',
+  'dispositions': 'Affirmed; Conflict Certified.',
?                           +

   'docket_numbers': '6D2023-1858',
   'download_urls': 'https://6dca.flcourts.gov/pre_opinion_content_download/2439523',
   'per_curiam': False,
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/idaho_civil_example.compare.json', module_string='juriscraper.opinions.united_states.state.idaho_civil')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[47 chars]trawn Lewis and Knight', 'download_urls': 'htt[193 chars]MRS'} != {'cas[47 chars]trawn; Lewis and Knight', 'download_urls': 'ht[194 chars]MRS'}
  {'blocked_statuses': False,
   'case_dates': '2014-03-19',
   'case_name_shorts': 'MRS',
-  'case_names': 'MRS v. Strawn Lewis and Knight',
+  'case_names': 'MRS v. Strawn; Lewis and Knight',
?                              +

   'date_filed_is_approximate': False,
   'docket_numbers': '40827',
   'download_urls': 'https://www.isc.idaho.gov/opinions/40827.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/lactapp_1_example.compare.json', module_string='juriscraper.opinions.united_states.state.lactapp_1')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'case_names': "Donald J. Rouse, Jr., Donald J. [538 chars]pdf'} != {'case_dates': '2019-11-12', 'case_names': "Dona[539 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2019-11-12',
   'case_name_shorts': '',
   'case_names': 'Donald J. Rouse, Jr., Donald J. Rouse, Sr., Thomas B. Rouse, '
                 "Allison Rouse Royster, and Rouse's Enterprises, LLC v. K&F "
                 "Restaurant Holdings, Ltd. d/b/a Izzo's Illegal Burrito, K&F "
                 'Restaurant Operations, LLC, G&O Pizza Holdings, Ltd. d/b/a Lit '
-                'Pizza, G&O Restaurant Operations, LLC and Osvaldo Fernandez',
+                'Pizza, G&O Restaurant Operations, LLC; and Osvaldo Fernandez',
?                                                      +

   'date_filed_is_approximate': False,
   'docket_numbers': '2019CW0586',
   'download_urls': '/opiniongrid/opinionpdf/2019%20CW%200586%20Decision%20Rehearing.pdf',
   'precedential_statuses': 'Unknown'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/mass_example.compare.json', module_string='juriscraper.opinions.united_states.state.mass')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[63 chars]neral Page v. Attorney General', 'download_url[244 chars]: ''} != {'cas[63 chars]neral; Page v. Attorney General', 'download_ur[245 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2024-06-27',
   'case_name_shorts': '',
-  'case_names': 'Schneider v. Attorney General Page v. Attorney General',
+  'case_names': 'Schneider v. Attorney General; Page v. Attorney General',
?                                              +

   'date_filed_is_approximate': False,
   'docket_numbers': 'SJC 13609 & 13610',
   'download_urls': '/doc/schneider-v-attorney-general-page-v-attorney-general-13609-13610/download',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/masslandct_example.compare.json', module_string='juriscraper.opinions.united_states.state.masslandct')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[86 chars]Trust v. Kettle Brook Lofts, LLC', 'download_u[230 chars]: ''} != {'cas[86 chars]Trust; Kettle Brook Lofts v. Kettle Brook Loft[274 chars]: ''}
Diff is 1075 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/md_example.compare.json', module_string='juriscraper.opinions.united_states.state.md')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[51 chars]GEICO Morse v. Erie Insurance', 'download_urls[268 chars]: ''} != {'cas[51 chars]GEICO; Morse v. Erie Insurance', 'download_url[269 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2015-05-27',
   'case_name_shorts': '',
-  'case_names': 'Woznicki v. GEICO Morse v. Erie Insurance',
+  'case_names': 'Woznicki v. GEICO; Morse v. Erie Insurance',
?                                  +

   'citations': '443 Md. 93',
   'date_filed_is_approximate': False,
   'docket_numbers': '52/14',
   'download_urls': '/opinions/coa/2015/52a14.pdf',
   'judges': 'Greene',
   'other_dates': '',
   'per_curiam': False,
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/minnag_example.compare.json', module_string='juriscraper.opinions.united_states.state.minnag')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[540 chars] 1, 205A.13 (2006) Minn. R. ch. 8205', 'case_name_shorts': ''} != {'cas[540 chars] 1, 205A.13 (2006); Minn. R. ch. 8205', 'case_name_shorts': ''}
  {'blocked_statuses': False,
   'case_dates': '2007-06-15',
   'case_name_shorts': '',
   'case_names': 'Op. Atty. Gen 185b (Cr. Ref. 159a-3)',
   'date_filed_is_approximate': False,
   'download_urls': 'tests/examples/opinions/united_states/185b-20070615.pdf',
   'precedential_statuses': 'Published',
   'summaries': 'SCHOOL ELECTIONS: PETITIONS: Petition Rules promulgated by the '
                'Secretary of State generally apply to petition for school '
                'district referendum. School district clerks should perform the '
                'functions of "filing officer under those rules:" Minn Stat. §§ '
-               '204B.071, 205A.05, subd. 1, 205A.13 (2006) Minn. R. ch. 8205'}
+               '204B.071, 205A.05, subd. 1, 205A.13 (2006); Minn. R. ch. 8205'}
?                                                          +


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/minnag_example_2.compare.json', module_string='juriscraper.opinions.united_states.state.minnag')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[408 chars]ition strong argument can be made president of[270 chars]: ''} != {'cas[408 chars]ition; strong argument can be made president o[272 chars]: ''}
Diff is 1163 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/minnctapp_u_example.compare.json', module_string='juriscraper.opinions.united_states.state.minnctapp_u')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[604 chars] here (2) even if the presumption applied, app[541 chars]: ''} != {'cas[604 chars] here; (2) even if the presumption applied, ap[544 chars]: ''}
Diff is 1865 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/mont_example.compare.json', module_string='juriscraper.opinions.united_states.state.mont')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[357 chars] Time Appellant is granted an extension of tim[132 chars]: ''} != {'cas[357 chars] Time; Appellant is granted an extension of ti[133 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2020-02-27',
   'case_name_shorts': '',
   'case_names': 'State v. W. Johnson',
   'date_filed_is_approximate': False,
   'docket_numbers': 'DA 19-0388',
   'download_urls': 'https://juddocumentservice.mt.gov/getDocByCTrackId?DocId=305504',
   'nature_of_suit': 'Direct Appeal',
   'precedential_statuses': 'Unpublished',
-  'summaries': 'Order - Grant - Extension of Time Appellant is granted an '
+  'summaries': 'Order - Grant - Extension of Time; Appellant is granted an '
?                                                 +

                'extension of time to and including April 6, 2020, within which '
                "to prepare, file, and serve Appellant's opening brief on "
                'appeal.'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/nc_example.compare.json', module_string='juriscraper.opinions.united_states.state.nc')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[333 chars]ppeal whether the trial court erred in denying[100 chars]: ''} != {'cas[333 chars]ppeal; whether the trial court erred in denyin[101 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2014-04-11',
   'case_name_shorts': '',
   'case_names': "Beroth Oil Co. v. N.C. Dep't of Transp.",
   'citations': '',
   'date_filed_is_approximate': False,
   'docket_numbers': '390PA11-2',
   'download_urls': 'http://appellate.nccourts.org/opinions/?c=1&pdf=31511',
   'precedential_statuses': 'Published',
-  'summaries': 'Interlocutory appeal whether the trial court erred in denying '
+  'summaries': 'Interlocutory appeal; whether the trial court erred in denying '
?                                    +

                "plaintiffs' motion for class certification of their inverse "
                'condemnation claim'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ncctapp_example_2013.compare.json', module_string='juriscraper.opinions.united_states.state.ncctapp')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[313 chars]trust reappointment of trustee.', 'case_name_shorts': 'McCrea'} != {'cas[313 chars]trust; reappointment of trustee.', 'case_name_[13 chars]rea'}
  {'blocked_statuses': False,
   'case_dates': '2013-12-17',
   'case_name_shorts': 'McCrea',
   'case_names': 'THZ Holdings, LLC v. McCrea',
   'citations': '',
   'date_filed_is_approximate': False,
   'docket_numbers': '13-425',
   'download_urls': 'http://appellate.nccourts.org/opinions/?c=2&pdf=30605',
   'precedential_statuses': 'Published',
-  'summaries': 'Breach of trust reappointment of trustee.'}
+  'summaries': 'Breach of trust; reappointment of trustee.'}
?                               +


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/nh_p_example.compare.json', module_string='juriscraper.opinions.united_states.state.nh_p')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[44 chars] E.R. In re H.R.', 'download_urls': 'https://w[262 chars]: ''} != {'cas[44 chars] E.R.; In re H.R.', 'download_urls': 'https://[263 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2023-06-29',
   'case_name_shorts': '',
-  'case_names': 'In re E.R. In re H.R.',
+  'case_names': 'In re E.R.; In re H.R.',
?                           +

   'citations': '',
   'date_filed_is_approximate': False,
   'docket_numbers': '2022-0543, 2022-0546',
   'download_urls': 'https://www.courts.nh.gov/sites/g/files/ehbemt471/files/documents/2023-06/2023033er-hr.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/nj_example.compare.json', module_string='juriscraper.opinions.united_states.state.nj')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[584 chars]dians they require a careful balancing of comp[366 chars]: ''} != {'cas[584 chars]dians; they require a careful balancing of com[367 chars]: ''}
Diff is 1377 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/njsuperctappdiv_p_example.compare.json', module_string='juriscraper.opinions.united_states.state.njsuperctappdiv_p')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[2431 chars]xpert and whether, under the current circumsta[362 chars]cpp'} != {'cas[2431 chars]xpert; and whether, under the current circumst[363 chars]cpp'}
Diff is 3774 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/njtaxct_p_example.compare.json', module_string='juriscraper.opinions.united_states.state.njtaxct_p')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[472 chars]nship Docket Nos. 008224-2022 & 008229-2022, o[1172 chars]: ''} != {'cas[472 chars]nship; Docket Nos. 008224-2022 & 008229-2022, [1176 chars]: ''}
Diff is 2664 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/nyappterm_2nd_example_2.compare.json', module_string='juriscraper.opinions.united_states.state.nyappterm_2nd')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[82 chars]enant Lilema Hamlet, Nonparty.', 'download_url[276 chars]: ''} != {'cas[82 chars]enant; Lilema Hamlet, Nonparty.', 'download_ur[277 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2017-08-29',
   'case_name_shorts': '',
-  'case_names': '15 Crown St. Realty, LLC v. Sonia Walker, Tenant Lilema '
+  'case_names': '15 Crown St. Realty, LLC v. Sonia Walker, Tenant; Lilema '
?                                                                 +

                 'Hamlet, Nonparty.',
   'citations': '',
   'date_filed_is_approximate': False,
   'docket_numbers': '',
   'download_urls': 'http://www.nycourts.gov/reporter/motions/2017/2017_85379.htm',
   'parallel_citations': '2017 NYSlipOp 85379(U)',
   'precedential_statuses': 'Unpublished'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_1_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_1')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[752 chars]omise rather, because the claim was based on a[134 chars]ott'} != {'cas[752 chars]omise; rather, because the claim was based on [135 chars]ott'}
Diff is 1283 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_2_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_2')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[265 chars]29209 29210', 'judges': 'Tucker', 'citations':[431 chars].P.'} != {'cas[265 chars]29209; 29210', 'judges': 'Tucker', 'citations'[432 chars].P.'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-22',
   'case_name_shorts': 'In re C.P.',
   'case_names': 'In re C.P.',
   'citations': '2021-Ohio-4504',
   'date_filed_is_approximate': False,
-  'docket_numbers': '29209 29210',
+  'docket_numbers': '29209; 29210',
?                          +

   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/2/2021/2021-Ohio-4504.pdf',
   'judges': 'Tucker',
   'precedential_statuses': 'Published',
   'summaries': 'The trial court awarded permanent custody of two children to a '
                'children services agency. Appellants, Father and Mother, assert '
                'the trial court erred when it concluded that it was in the '
                "children's best interest to grant permanent custody to the "
                'agency. The record contains compelling evidence supporting the '
                "trial court's best interest conclusion. Judgments affirmed."}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_3_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_3')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[268 chars]21-12 17-21-13', 'judges': 'Willamowski', 'cit[407 chars].R.'} != {'cas[268 chars]21-12; 17-21-13', 'judges': 'Willamowski', 'ci[408 chars].R.'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-20',
   'case_name_shorts': 'In re K.R.',
   'case_names': 'In re K.R.',
   'citations': '2021-Ohio-4474',
   'date_filed_is_approximate': False,
-  'docket_numbers': '17-21-12 17-21-13',
+  'docket_numbers': '17-21-12; 17-21-13',
?                             +

   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/3/2021/2021-Ohio-4474.pdf',
   'judges': 'Willamowski',
   'precedential_statuses': 'Published',
   'summaries': 'COVID protocols did not prevent appellant from working the case '
                'plan when she had over a year to work it before COVID protocols '
                'were enacted and did not do so. The determination of the trial '
                'court to grant permanent custody to the Agency was not against '
                'the manifest weight of the evidence and was in the best '
                'interest of the children.'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_4_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_4')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[343 chars]. 201 judicial notice Civ.R. 61 harmless error[96 chars].B.'} != {'cas[343 chars]. 201; judicial notice; Civ.R. 61; harmless er[102 chars].B.'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-17',
   'case_name_shorts': 'In re Lu.B.',
   'case_names': 'In re Lu.B.',
   'citations': '2021-Ohio-4479',
   'date_filed_is_approximate': False,
   'docket_numbers': '21CA1',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/4/2021/2021-Ohio-4479.pdf',
   'judges': 'Hess',
   'precedential_statuses': 'Published',
-  'summaries': 'Evid.R. 201 judicial notice Civ.R. 61 harmless error legal '
+  'summaries': 'Evid.R. 201; judicial notice; Civ.R. 61; harmless error; legal '
?                           +                +          +               +

-               'custody R.C. 2151.353(A)(3) best interest of the children'}
+               'custody; R.C. 2151.353(A)(3); best interest of the children'}
?                       +                    +


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_5_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_5')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[360 chars] contrary to law res judicata', 'case_name_shorts': 'Lindsay'} != {'cas[360 chars] contrary to law; res judicata', 'case_name_shorts': 'Lindsay'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-22',
   'case_name_shorts': 'Lindsay',
   'case_names': 'State v. Lindsay',
   'citations': '2021-Ohio-4526',
   'date_filed_is_approximate': False,
   'docket_numbers': '2021 CA 0068',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/5/2021/2021-Ohio-4526.pdf',
   'judges': 'J. Wise',
   'precedential_statuses': 'Published',
-  'summaries': 'Sentence contrary to law res judicata'}
+  'summaries': 'Sentence contrary to law; res judicata'}
?                                        +


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_6_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_6')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[510 chars]Jones R.C. 2929.11 and 2929.12 Beasley R.C. 29[52 chars]ord'} != {'cas[510 chars]Jones; R.C. 2929.11 and 2929.12; Beasley; R.C.[57 chars]ord'}
  {'blocked_statuses': False,
   'case_dates': '2021-11-19',
   'case_name_shorts': 'Ford',
   'case_names': 'State v. Ford',
   'citations': '2021-Ohio-4403',
   'date_filed_is_approximate': False,
   'docket_numbers': 'WD-20-085',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/6/2021/2021-Ohio-4403.pdf',
   'judges': 'Osowik',
   'precedential_statuses': 'Published',
   'summaries': "Appellant's gross sexual imposition convictions are not allied "
                'and subject to merger and his consecutive sentences are not '
-               'clearly and convincingly contrary to law. Jones R.C. 2929.11 '
+               'clearly and convincingly contrary to law. Jones; R.C. 2929.11 '
?                                                               +

-               'and 2929.12 Beasley R.C. 2929.14(C) Ruff R.C. 2945.21.'}
+               'and 2929.12; Beasley; R.C. 2929.14(C); Ruff; R.C. 2945.21.'}
?                           +        +                +     +


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_7_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_7')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[366 chars]ldren conviction not against the manifest weig[46 chars]ins'} != {'cas[366 chars]ldren; conviction not against the manifest wei[47 chars]ins'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-27',
   'case_name_shorts': 'Wiggins',
   'case_names': 'State v. Wiggins',
   'citations': '2021-Ohio-4554',
   'date_filed_is_approximate': False,
   'docket_numbers': '20 MA 0125',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/7/2021/2021-Ohio-4554.pdf',
   'judges': 'Donofrio',
   'precedential_statuses': 'Published',
-  'summaries': 'endangering children conviction not against the manifest weight '
?                                                                         -------

+  'summaries': 'endangering children; conviction not against the manifest '
?                                    +

-               'of the evidence'}
+               'weight of the evidence'}
?                +++++++


======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_8_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_8')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[382 chars]gment de novo workers' compensation R.C. 4123.[612 chars]son'} != {'cas[382 chars]gment; de novo; workers' compensation; R.C. 41[623 chars]son'}
Diff is 1955 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_9_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_9')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[371 chars]uence traffic violation, field sobriety testin[47 chars]Won'} != {'cas[371 chars]uence; traffic violation, field sobriety testi[48 chars]Won'}
  {'blocked_statuses': False,
   'case_dates': '2021-11-01',
   'case_name_shorts': 'Won',
   'case_names': 'State v. Won',
   'citations': '2021-Ohio-3869',
   'date_filed_is_approximate': False,
   'docket_numbers': '20CA0076-M',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/9/2021/2021-Ohio-3869.pdf',
   'judges': 'Carr',
   'precedential_statuses': 'Published',
-  'summaries': 'driving while under the influence traffic violation, field '
+  'summaries': 'driving while under the influence; traffic violation, field '
?                                                 +

                'sobriety testing, reasonable suspicion'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_10_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_10')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[541 chars]e evidence judgment affirmed.", 'case_name_shorts': 'Gysegem'} != {'cas[541 chars]e evidence; judgment affirmed.", 'case_name_shorts': 'Gysegem'}
  {'blocked_statuses': False,
   'case_dates': '2021-12-21',
   'case_name_shorts': 'Gysegem',
   'case_names': 'Gysegem v. Ohio State Univ. Wexner Med. Ctr.',
   'citations': '2021-Ohio-4496',
   'date_filed_is_approximate': False,
   'docket_numbers': '20AP-477',
   'download_urls': 'https://www.supremecourt.ohio.gov/rod/docs/pdf/10/2021/2021-Ohio-4496.pdf',
   'judges': 'Brown',
   'precedential_statuses': 'Published',
   'summaries': 'Judgment of Court of Claims finding in favor of appellee on '
                "appellants' claims for medical negligence and loss of "
-               'consortium was not against the manifest weight of the evidence '
+               'consortium was not against the manifest weight of the evidence; '
?                                                                              +

                'judgment affirmed.'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_11_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_11')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[369 chars]s Law Senate Bill 231 R.C. 2903.41-44 Violent [229 chars]: ''} != {'cas[369 chars]s Law; Senate Bill 231; R.C. 2903.41-44; Viole[235 chars]: ''}
Diff is 1315 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctapp_12_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctapp_12')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[280 chars]6-017 CA2021-06-018', 'judges': 'M. Powell', '[358 chars]lip'} != {'cas[280 chars]6-017; CA2021-06-018', 'judges': 'M. Powell', [360 chars]lip'}
Diff is 1016 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ohioctcl_beginningofyear_example.compare.json', module_string='juriscraper.opinions.united_states.state.ohioctcl_beginningofyear')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[389 chars]ation tortious spoliation of documents. Review[2149 chars]ter'} != {'cas[389 chars]ation; tortious spoliation of documents. Revie[2152 chars]ter'}
Diff is 3709 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/okla_example.compare.json', module_string='juriscraper.opinions.united_states.state.okla')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[1058 chars]l law and 2) the district court's exclusive ju[297 chars]HIS'} != {'cas[1058 chars]l law; and 2) the district court's exclusive j[300 chars]HIS'}
Diff is 2605 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/oklacrimapp_example.compare.json', module_string='juriscraper.opinions.united_states.state.oklacrimapp')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[570 chars]43.5 and Count 2: Child Sexual Abuse -- Victim[191 chars]GER'} != {'cas[570 chars]43.5 ; and Count 2: Child Sexual Abuse -- Vict[193 chars]GER'}
Diff is 1103 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/ri_example.compare.json', module_string='juriscraper.opinions.united_states.state.ri')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[1334 chars]ppeal the stay was vacated, and restitution pa[869 chars]: ''} != {'cas[1334 chars]ppeal; the stay was vacated, and restitution p[870 chars]: ''}
Diff is 3009 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/tenn_example.compare.json', module_string='juriscraper.opinions.united_states.state.tenn')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[431 chars]pbell Justice Jeffrey S. Bivins joins', 'lower[1185 chars]: ''} != {'cas[431 chars]pbell; Justice Jeffrey S. Bivins joins', 'lowe[1186 chars]: ''}
Diff is 2184 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/tennctapp_example.compare.json', module_string='juriscraper.opinions.united_states.state.tennctapp')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[1327 chars] 2018 (2) the Board's decision was supported b[150 chars]: ''} != {'cas[1327 chars] 2018; (2) the Board's decision was supported [152 chars]: ''}
Diff is 2166 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/tenncrimapp_example.compare.json', module_string='juriscraper.opinions.united_states.state.tenncrimapp')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[657 chars]unsel (2) he entered into his guilty plea unkn[465 chars]: ''} != {'cas[657 chars]unsel; (2) he entered into his guilty plea unk[471 chars]: ''}
Diff is 2121 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/va_example.compare.json', module_string='juriscraper.opinions.united_states.state.va')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[1283 chars]poses rather, he only argued that it was not a[198 chars]son'} != {'cas[1283 chars]poses; rather, he only argued that it was not [199 chars]son'}
Diff is 2045 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/vactapp_u_example.compare.json', module_string='juriscraper.opinions.united_states.state.vactapp_u')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[442 chars]irmed appellant's other arguments regarding Ju[63 chars]: ''} != {'cas[442 chars]irmed; appellant's other arguments regarding J[64 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2013-10-08',
   'case_name_shorts': '',
   'case_names': 'Theodis Thomas Smith, Jr. v. Commonwealth of Virginia',
   'date_filed_is_approximate': False,
   'dispositions': "Trial court's denial of appellant's motion to strike Juror "
                   '12 based on her friendship with a police officer who was a '
-                  "potential witness affirmed appellant's other arguments "
+                  "potential witness affirmed; appellant's other arguments "
?                                             +

                   'regarding Juror 12 waived',
   'docket_numbers': '1235123',
   'download_urls': 'tests/examples/opinions/united_states/opinions/opncavwp/1235123.pdf',
   'precedential_statuses': 'Unpublished'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/vactapp_p_example.compare.json', module_string='juriscraper.opinions.united_states.state.vactapp_p')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[440 chars]irmed appellant's other arguments regarding Ju[63 chars]: ''} != {'cas[440 chars]irmed; appellant's other arguments regarding J[64 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2013-10-08',
   'case_name_shorts': '',
   'case_names': 'Theodis Thomas Smith, Jr. v. Commonwealth of Virginia',
   'date_filed_is_approximate': False,
   'dispositions': "Trial court's denial of appellant's motion to strike Juror "
                   '12 based on her friendship with a police officer who was a '
-                  "potential witness affirmed appellant's other arguments "
+                  "potential witness affirmed; appellant's other arguments "
?                                             +

                   'regarding Juror 12 waived',
   'docket_numbers': '1235123',
   'download_urls': 'tests/examples/opinions/united_states/opinions/opncavwp/1235123.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_state_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/wyo_example.compare.json', module_string='juriscraper.opinions.united_states.state.wyo')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[253 chars], LLC Clark's Fork Canyon Springs, LLC and Sun[292 chars]: ''} != {'cas[253 chars], LLC; Clark's Fork Canyon Springs, LLC; and S[294 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2014-12-10',
   'case_name_shorts': '',
   'case_names': 'Philippe Lajaunie, individually and derivatively as a member '
                 'of Beartooth Mountain Springs, LLC and American Summits, LLC '
                 'v. Daren Singer, individually and in his capacity as a manager '
-                "of Beartooth Mountain Springs, LLC Clark's Fork Canyon "
+                "of Beartooth Mountain Springs, LLC; Clark's Fork Canyon "
?                                                   +

-                'Springs, LLC and Sunlight Springs Distribution, LLC',
+                'Springs, LLC; and Sunlight Springs Distribution, LLC',
?                             +

   'citations': '2014 WY 159',
   'date_filed_is_approximate': False,
   'docket_numbers': 'S-14-0005',
   'download_urls': 'https://documents.courts.state.wy.us/Opinions/2014WY159.pdf',
   'precedential_statuses': 'Published'}

======================================================================
FAIL: test_scrape_opinion_territories_example_files (tests.local.test_ScraperExampleTest.ScraperExampleTest) [Testing example files] (json_path='tests/examples/opinions/united_states/visuper_u_example.compare.json', module_string='juriscraper.opinions.united_states.territories.visuper_u')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "juriscraper/tests/local/test_ScraperExampleTest.py", line 126, in run_tests_on_module_str
    self.assertEqual(
AssertionError: {'cas[112 chars]lands Wynnie Testamark, Director, VI Bureau of[438 chars]: ''} != {'cas[112 chars]lands; Wynnie Testamark, Director, VI Bureau o[440 chars]: ''}
  {'blocked_statuses': False,
   'case_dates': '2023-09-20',
   'case_name_shorts': '',
   'case_names': 'Randy Burke v. Ariel M. Smith, Esquire, Attorney General of '
-                'the Virgin Islands Wynnie Testamark, Director, VI Bureau of '
+                'the Virgin Islands; Wynnie Testamark, Director, VI Bureau of '
?                                   +

-                'Corrections Dana Grant, Acting Warden, John a. Bell, '
+                'Corrections; Dana Grant, Acting Warden, John a. Bell, '
?                            +

                 'Correctional Facility',
   'citations': '2023 VI Super 57U',
   'date_filed_is_approximate': False,
   'docket_numbers': 'SX-2013-CV-377',
   'download_urls': 'https://superior.vicourts.org/UserFiles/Servers/Server_12810747/File/Opinions/2023/SX-2013-CV-00377-Documents '
                    '57U.pdf',
   'judges': 'Brady, Douglas A.',
   'precedential_statuses': 'Unpublished'}
```

Reply:
>  I did see one example above where the semi-colon was removed from the end of the string, and that does make sense to remove.

Actually on closer inspection that should not be removed either.  that semicolon is in the middle of the content we want to collect but the scraper is incorrectly cropping the data early.  

every example cited above suggests that the correct output is with a semicolon within the content.